new-frontiers-14/frontier-station-14
# Guidebook changes for new players

## About the PR
Made the guidebook automatically open on spawning for anyone with less than 3 hours of playtime on Frontier, added an additional guidebook page titled "Starting Equipment" detailing the items you get upon spawning for the first time (Traffic key, survival box, medical implant)
## Why / Balance
Discussion with Dvir relating to discord thread https://ptb.discord.com/channels/1123826877245694004/1319849208030953493
## How to test
Load up a server
Check the guidebook tab for the new page
Spawn in and see the guidebook open
## Media
![image](https://github.com/user-attachments/assets/5ed32df2-3b8a-4994-b8dc-b6a5fed45119)

## Requirements

## Breaking changes
None I am aware of
**Changelog**
N/A
